public/assets/analytics/track.js

```diff
@@ -1,6 +1,13 @@
-/*
```

Comment:
The piwik.js file was being overwritten to this location (track.js) by Dr Elephant MP wrapper. Since the wrapper will be deprecated so adding this file as part of Dr Elephant source code itself.